src/Bundle/test/src/Tests/Controller/XmlApiTestCase.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace App\Tests\Controller;
+
+use ApiTestCase\XmlApiTestCase as BaseXmlApiTestCase;
+use Symfony\Component\HttpFoundation\Response;
+
+abstract class XmlApiTestCase extends BaseXmlApiTestCase
+{
+    protected static array $headersWithContentType = [
```

Comment:
Should we open a PR with it to ApiTestCase?

Reply:
good idea, cause Symfony will return a **text/xml** as response